### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by andriy-shymkiv**
[DEMO LINK](https://andriy-shymkiv.github.io/react_todo-app-loading-todos/)

I will handle the rest of Errors later, in the next two tasks

### CODE REVIEW

**Review by Illia-Skladnik — APPROVED** [review]
1. Your Toggle all button in header doesn't work. It should be inactive of you got as minimum 1 active todo (as I understand)
(If you manually toggle all todos to completed state, “Toggle all” should stay active.)
2. status-lable also doesn't work. There should be check symbol in completed todos.
3. you shouldn't use user as a props I guess. You arleady have it in useContext. 
4. I'm not sure it is necessary, but you can create separate component for your errors block.

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  const handleFormSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    if (!query.length) {
+      onErrorChange(true);
+    }
+
+    setTimeout(() => {
+      onErrorChange(false);
+    }, 3000);
```
Comment (mykola-koval) [inline src/components/Header/Header.tsx:27]: let's clear this timeout when we close the error

File: src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        {`${activeTodos.length} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          data-cy="FilterLinkAll"
+          href="#/"
+          className={cn(
+            'filter__link',
+            { selected: clickedValue === 0 },
```
Comment (mykola-koval) [inline src/components/Footer/Footer.tsx:70]: this totally should be refactored

**Review by mykola-koval — APPROVED** [review]
you can implement fixes in the next task

**Pull request closed without merging**